ccd-definition/AuthorisationCaseEvent/DEPRECATED.json

```diff
@@ -335,5 +335,19 @@
     "CaseEventID": "uploadC2Gatekeeping",
     "UserRole": "[SOLICITOR]",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "amendChildren",
```

Comment:
what about events like amendChildrenGatekeeping? I think gatekeeper should be able to see old events which are deprecated now

Reply:
you are right, will add + check other permissions that may be missing